### DISCUSSION TITLE
Question 40:  In Taxonomy, how to handle excess taxon ranks? [CM-Geo]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1027] [Q&A]
Description: A specimen’s taxon hierarchy does not always fit neatly into the given EMu fields. What is everyone doing when they have more taxon terms than EMu fields? (i.e. If you have an Infraclass-rank term or unranked clade)

**Comment by cmaves3** [comment]
Response: [RB] Insects has similar data issues and data is currently put into notes. So the data is there but not in the hierarchy. 
	Insects Example Taxonomy IRNs - 573413 704979 705244

**Reply by rondlg** [reply]
A-ha ticket submitted by cmaves

EMu Idea is up!
https://emu.ideas.aha.io/ideas/EMU-I-126

**Comment by cmaves3** [comment]
ClaInfraclass already exists in the backend of our EMu - don't know if it can be enabled
